front-end/src/components/Body/Pagination/Pagenation.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import "./style.scss";
```

Comment:
Good Job and Thank you for your effort.

I think we do not need spent time to code the logic here.
Next lesson I will guide how to use the **Third-party** support for it.
Thanks